### ISSUE TITLE
Memory requirements in readme wrong

### ISSUE THREAD

**Issue by grobalt** [madMAx43v3r/chia-plotter#194]
With 16 threads it consumes way more than 6GB
![image](https://user-images.githubusercontent.com/79084968/121603587-47259200-ca49-11eb-9ad4-bdd56cf905dd.png)

**Comment by madMAx43v3r** [comment]
yeah that part is outdated, I increased the number of read threads some time

**Comment by grobalt** [comment]
thanks .. currently testing the limits with 128GB ... 16 was working, 32 failed .. 24 failed as well

**Comment by Hardcore-fs** [comment]
it depends on the os.......

![process](https://user-images.githubusercontent.com/1309004/121606553-efa31880-ca80-11eb-9473-3d03addbf01f.jpeg)

![process2](https://user-images.githubusercontent.com/1309004/121606557-f29e0900-ca80-11eb-81ea-8b82acda42d9.jpeg)

**Comment by madMAx43v3r** [comment]
yeah others have found 16 threads with 128GB to be the max also

**Comment by grobalt** [comment]
> yeah others have found 16 threads with 128GB to be the max also

thanks .. then 2048gb is the correct sizing for dual socket 64core cpus, resulting in 8 parallel plots finished in ~ 22 minutes.

**Comment by Hardcore-fs** [comment]
I suspect GPU's will shit all over that figure...
but the guys over at  Blake3 will be ecstatic......

**Comment by vvavepacket** [comment]
@grobalt  whats 128GB, the system total mqemory or just the ramdisk for temp drives?

**Comment by grobalt** [comment]
total, for ramdisk i used ramfs, so no fixed size
